File: Java/Example5/ExampleClass.java
package Example5;

public class ExampleClass implements IExample {
    private String name;
    private int age;

    // Constructor con múltiples parámetros
    public ExampleClass(String name, int age) {
        this.name = name;
        this.age = age;
    }

    // Implementación de interfaz - cuando la interfaz es implementada
    @Override
    public void performAction() {
        System.out.println("Accion realizada en Example5");
    }
}

File: CSharp/Example5/ExampleClass.cs
namespace Example5
{
    public class ExampleClass : IExample
    {
        public string Name { get; set; }
        public int Age { get; set; }

        // Constructor con múltiples parámetros
        public ExampleClass(string name, int age)
        {
            Name = name;
            Age = age;
        }

        // Implementación de interfaz - cuando la interfaz es implementada
        public void PerformAction()
        {
            Console.WriteLine("Accion realizada en Example5");
        }
    }
}